client/utils/utils.go

```diff
@@ -28,7 +29,7 @@ func SendTx(txCtx authctx.TxContext, cliCtx context.CLIContext, msgs []sdk.Msg)
 		if err != nil {
 			return err
 		}
-		fmt.Fprintf(os.Stdout, "estimated gas = %v\n", txCtx.Gas)
+		fmt.Fprintf(os.Stderr, "estimated gas = %v\n", txCtx.Gas)
```

Comment:
Shouldn't it be standard output instead of standard error ?

Reply:
As per UNIX's philosophy and specs, errors and diagnostics should go to stderr. stdout is where you want to write the desired output of a program's operation.